GliaWidgets/Sources/CallVisualizer/CallVisualizer.swift

```diff
@@ -92,6 +91,7 @@ public final class CallVisualizer {
 extension CallVisualizer {
     func handleAcceptedUpgrade() {
         environment.log.prefixed(Self.self).info("Incoming Call Visualizer engagement auto accepted")
+        delegate?(.visitorCodeIsRequested)
```

Comment:
I guess previously `.visitorCodeIsRequested` action was a trigger for Interactor creation. Let's check if it can be removed completely

Reply:
Good idea